### ISSUE TITLE
Jax vs normal agent

### ISSUE THREAD

**Issue by satst27** [google/dopamine#153]
Hi, 

I think it will helpful to clarify that the checkpoints that you have provided are for the Jax verion and not the normal version:

"The TensorFlow checkpoint files for 5 independent runs of the 4 agents on all 60 games are available. The format for each of the files is: https://storage.cloud.google.com/download-dopamine-rl/lucid/${AGENT}/${GAME}/${RUN}/tf_ckpt-199.${SUFFIX}, where:

    AGENT can be "dqn", "c51", "rainbow", or "iqn"."

Are there checkpoints available for all games for non-Jax version as well?

thanks,
satst27

**Comment by psc-g** [comment]
hi, the checkpoints are for the tensorflow version, not for the jax
version. we don't have checkpoints for all games and all agents on JAX, but
i'm hoping to be able to release the quantile checkpoints on JAX sometime
in the near future (since quantile is only available on JAX).

**Comment by houeland** [comment]
Bump, is this actually completed? I see that e.g. https://console.cloud.google.com/storage/browser/_details/download-dopamine-rl/jax/quantile/Pong/1/ckpt.199 exists now.

(I currently only see 6 games there, but at least it's not 0. download-dopamine-rl/jax/dqn_adam_mse looks like it actually has checkpoints for all the trained games(?), while download-dopamine-rl/jax/iqn has 0.)

**Comment by psc-g** [comment]
sorry for delay, will get to this after the iclr deadline!
the dqn_adam_mse folder does have for all games, but as the directory
suggests it is with Adam optimizer and MSE loss (as we explored in
Revisiting Rainbow: http://proceedings.mlr.press/v139/ceron21a.html)

we haven't released checkpoints externally for all games for the jax
agents. after the iclr deadline i'll see about releasing all of them.
